Analytics/App_Plugins/Analytics/backOffice/AnalyticsTree/controllers/Settings.Controller.js

```diff
@@ -106,8 +106,12 @@
                 //Don't need anything from response.data back
 
                 //Display Success message
-                notificationsService.success("Success profile details have been saved");
+                notificationsService.success(localizationService.localize("analytics_profileDetailsSaved"));
             });
+
+            //Sync ('refresh') the tree!
+            //This should work, but somehow it seems to fails sometimes when you click "save" and receive an error in a dialog on the right.
+            //navigationService.syncTree({ tree: 'analyticsTree', path: [-1, -1], forceReload: false, activate: true });
```

Comment:
@warrenbuckley yes, it is also just translations - but while I changed the translations of the trees and it settings, I think it would be cool to refresh the trees after click on save (to show/hide eCommerce node based on there is any transactions in Google Analytics). But it seems to have some issues, so I have added a refresh action on root node, so you can refresh trees without refreshing entire page :)